bedrock/firefox/templates/firefox/nightly/firstrun.html

```diff
@@ -26,17 +26,12 @@
 
 {% block site_header %}{% endblock %}
 
-{% if switch('nightly-doge') %}
-  {% set hero_class='mzp-t-dark mzp-t-product-nightly main-header is-doge' %}
-{% else %}
-  {% set hero_class='mzp-t-dark mzp-t-product-nightly main-header' %}
-{% endif %}
 
 {% block content %}
 <main role="main">
   {{ hero(
     title=ftl('nightly-firstrun-thank-you-for-using'),
-    class=hero_class,
+    class='mzp-t-dark mzp-t-product-nightly main-header,
```

Comment:
Missing closing quote here
```suggestion
    class='mzp-t-dark mzp-t-product-nightly main-header',
```

Reply:
D'oh! Good catch. Serves me right for not reloading the page.